Mathlib/FieldTheory/RatFunc.lean

```diff
@@ -781,9 +788,7 @@ theorem liftRingHom_injective [Nontrivial R] (φ : R[X] →+* L) (hφ : Function
 
 end LiftHom
 
-variable (K)
```

Comment:
Where did this go?

Reply:
I also wondered about this: the binder of the variable `K` has been updated inside a `section`, so it did not affect the binder of the `K` that is in scope here.

Reply:
I think this is caused by the mis-port in the comment below.